Qt/mainwindow.cpp

```diff
@@ -80,48 +80,66 @@ void MainWindow::newFrame()
 
 void MainWindow::updateMenus()
 {
-	foreach(QAction * action, anisotropicGroup->actions()) {
-		if (g_Config.iAnisotropyLevel == action->data().toInt()) {
+	foreach(QAction * action, windowGroup->actions()) {
+		int width = (g_Config.IsPortrait() ? 272 : 480) * action->data().toInt();
+		int height = (g_Config.IsPortrait() ? 480 : 272) * action->data().toInt();
+		if (g_Config.iWindowWidth == width && g_Config.iWindowHeight == height) {
 			action->setChecked(true);
 			break;
 		}
 	}
 
-	foreach(QAction * action, screenGroup->actions()) {
-		int width = (g_Config.IsPortrait() ? 272 : 480) * action->data().toInt();
-		int height = (g_Config.IsPortrait() ? 480 : 272) * action->data().toInt();
-		if (g_Config.iWindowWidth == width && g_Config.iWindowHeight == height) {
+	foreach(QAction * action, renderingResolutionGroup->actions()) {
```

Comment:
nit: Could we make this a method?  Something like...

```c++
void MainWindow::updateMenuGroupInt(QActionGroup *group, int value) {
	foreach (QAction *action, group->actions()) {
		action->setChecked(action->data().toInt() == value);
	}
}

updateMenuGroupInt(renderingResolutionGroup, g_Config.iInternalResolution);
updateMenuGroupInt(renderingModeGroup, g_Config.iRenderingMode);
```

Disclaimer: I don't really know Qt slots/etc. well.  Not sure if it's better to only set true on the one matching one?

There's a lot of int settings, so this would probably make the code more readable and easier to add/remove settings later.

-[Unknown]

Reply:
It doesn't compile and shows something like `error: prototype for ‘void MainWindow::updateMenuGroupInt(QActionGroup*, int)’ does not match any in class ‘MainWindow’`.

Reply:
Hm, it sounds like you might've skipped adding it to the header?  But no big deal anyway.  I think it's fine.

-[Unknown]